web/docs/platform-cli.md

```diff
@@ -111,15 +111,31 @@ The platform performs a POST request against this URL when the alert triggers.
 #### `<webhook_body>`
 
 The body to send along with the webhook. Must be valid JSON. The body may
-contain the string `$NODE_ID`, which will be replaced by the node id that
-triggered the alert.
+contain the string `$NODE_NAME`, which will be replaced by the name of the
+node that triggered the alert.
 
-Defaults to `{"text": "Node {node_id} disconnected for more than {duration}s"}`,
+Defaults to `{"text": "Node $NODE_NAME disconnected for more than {duration}s"}`,
 where `node_id` and `duration` are set dynamically from the CLI parameters.
 
-#### `<node_name>`
+### Example
```

Comment:
Should this be a _tutorial_ instead?

Reply:
I misunderstood the comment, I thought this was just suggesting to rename the heading from `Example` to `Tutorial`. Instead, the idea was to move it to the top-level tutorial section of the website.

We agreed to leave it as is for now, and ask @mavam for feedback on the location.

Reply:
After talking to @mavam he is suggesting Guides/Usage as the location.